pyani_plus/private_cli.py

```diff
@@ -34,41 +34,76 @@
 
 from pyani_plus import db_orm, tools
 from pyani_plus.methods import method_anib, method_anim, method_dnadiff, method_fastani
+from pyani_plus.public_cli import (
+    OPT_ARG_TYPE_CREATE_DB,
+    OPT_ARG_TYPE_FRAGSIZE,
+    OPT_ARG_TYPE_KMERSIZE,
+    OPT_ARG_TYPE_MINMATCH,
+    REQ_ARG_TYPE_DATABASE,
+    REQ_ARG_TYPE_OUTDIR,
+    REQ_ARG_TYPE_RUN_NAME,
+)
 from pyani_plus.utils import file_md5sum
 
 app = typer.Typer()
 
-
-@app.command()
+REQ_ARG_TYPE_FASTA_FILES = Annotated[
```

Comment:
Does it make any sense to abstract these out into a companion "header/`.h`" - like file, or is it better to keep them close to the code here do you think?